config/config.go

```diff
@@ -32,9 +32,10 @@ type (
 		}
 
 		Pool struct {
-			Min    int           `default:"2"`
-			Max    int           `default:"4"`
-			MinAge time.Duration `default:"55m" split_words:"true"`
+			Min              int           `default:"2"`
+			Max              int           `default:"4"`
+			MinAge           time.Duration `default:"55m" split_words:"true"`
+			ExistingCapacity int           `envconfig:"DRONE_POOL_EXISTING_CAPACITY" default:"0"`
```

Comment:
You should use `split_words` instead of enforcing the env variable:

```diff
- 			ExistingCapacity int           `envconfig:"DRONE_POOL_EXISTING_CAPACITY" default:"0"`
+ 			ExistingCapacity int           `default:"0" split_words:"true"`
```

Reply:
Thanks. I've just updated.